pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -380,37 +380,129 @@ func (c *Cluster) initMonIPs(mons []*monConfig) error {
 	return nil
 }
 
+// find a suitable node for this monitor. first look for a fault zone that has
+// no monitors. only after the nodes with zone labels are considered are the
+// unlabeled nodes considered. reasonsing: without more information, _at worst_
+// the unlabled nodes are actually in the set of labeled zones, and at best,
+// they are in distinct zones.
+func scheduleMonitor(mon *monConfig, nodeZones [][]NodeUsage) *NodeUsage {
+	// the node choice for this monitor
+	var nodeChoice *NodeUsage = nil
```

Comment:
FWIW My linter complains if block ends with a return statement, so drop this else and outdent its block

Reply:
fixed